packages/cozy-konnector-libs/src/helpers/cozy-client-js-stub.js

```diff
@@ -56,10 +56,14 @@ module.exports = {
       setDefaults(doctype)
       const { selector } = options
       const keys = Object.keys(selector)
-      const result = db
+      let result = db
         .get(doctype)
         .filter(doc => keys.every(key => doc[key]))
         .value()
+
+      if (options.wholeResponse) {
+        result = { docs: result }
```

Comment:
Reusing a variable name is a smell in a function's code that responsibilities are not well established.
```
db
        .get(doctype)
        .filter(doc => keys.every(key => doc[key]))
        .value()
```
may be a heavy processing, and never be used if `option.wholeResponse` is `true`.

A `if/then` seems a better solution here.